src/blr/cli/bamtofq.py

```diff
@@ -0,0 +1,96 @@
+"Writes fq files from bam files"
```

Comment:
```suggestion
"Writes paired or interleaved FASTQ from BAM file."
```

Reply:
Since it doesn't write interleaved I'm skipping this.